pennylane/pulse/rydberg.py

```diff
@@ -114,6 +115,140 @@ def circuit():
     return HardwareHamiltonian(coeffs, observables, settings=settings)
 
 
+def rydberg_drive(amplitude, phase, detuning, wires):
+    r"""Returns a :class:`ParametrizedHamiltonian` representing the action of a driving laser
+    field with the given amplitude, detuning and phase acting on the given wires
+
+    .. math::
+
+        \frac{1}{2} \Omega(t) \sum_{i \in \text{wires}} (\cos(\phi)\sigma_i^x - \sin(\phi)\sigma_i^y) -
+        \delta(t) \sum_{i \in \text{wires}} \sigma_i^z
```

Comment:
```suggestion
        \frac{1}{2} \Omega(t) \sum_{q \in \text{wires}} (\cos(\phi)\sigma_q^x - \sin(\phi)\sigma_q^y) -
        \delta(t) \sum_{q \in \text{wires}} \sigma_q^z
```
Optional, this is to be consistent with ``drive``